### ISSUE TITLE
Fix .htaccess file to allow favicon and fonts from theme to 

### ISSUE THREAD

**Issue by kehh** [wamuseum/providence#151]
This is low priority but it just adds to the logs unneccessarily and also people may actually need the font / favicon.

```
[Thu Jan 08 11:07:06.130506 2015] [access_compat:error] [pid 30482] [client 192.168.57.177:39543] AH01797: client denied by server configuration: /data/cmis/collectiveaccess/providence/current/favicon.ico
[Thu Jan 08 11:07:25.637164 2015] [access_compat:error] [pid 29782] [client 192.168.57.191:53075] AH01797: client denied by server configuration: /data/cmis/collectiveaccess/providence/current/themes/default/css/fonts/28EC2C_0_0.eot, referer: http://wamcmis.dca.wa.gov.au/index.php/find/SearchObjects/Index
```

**Comment by leftclickben** [comment]
This is a bug not an enhancement :-)

**Comment by leftclickben** [comment]
Closed already by #173
